pkg/store/employee/filter.go

```diff
@@ -13,3 +15,10 @@ type EditGeneralInfo struct {
 	GithubID      string
 	NotionID      string
 }
+
+type EditSkills struct {
```

Comment:
This name may be confused with a function do edit skill. Input names should end with a fixed suffix (like `filter`, `input`, `request`,...)

Reply:
thank you, it's done